locale/ja/editor.po

```diff
@@ -27,7 +26,7 @@ msgid "editor.submission.workflowDecision.submission"
 msgstr "決定を記録しました"
 
 msgid "editor.submission.workflowDecision.submission.declined"
-msgstr "投稿は不採用されました。"
```

Comment:
There seem to changes/removal unrelated to this PR here. Did these get committed accidentally?